tests/Guzzle5ExceptionTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Http\Adapter\Tests;
+
+use GuzzleHttp\Exception as GuzzleExceptions;
+use Http\Adapter\Guzzle5HttpAdapter;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class Guzzle5ExceptionTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetException()
+    {
+        $request = $this->getMock('Psr\Http\Message\RequestInterface');
```

Comment:
I don't think we need to mock HTTP Message objects. Those are Value Objects, you are free to use them here.